rotkehlchen/tests/external_apis/test_coingecko.py

```diff
@@ -63,3 +68,64 @@ def test_assets_with_icons(icon_manager):
     """Checks that _assets_with_coingecko_id returns a proper result"""
     x = icon_manager._assets_with_coingecko_id()
     assert len(x) > 1000
+
+
+def test_asset_icons_for_collections(icon_manager: IconManager) -> None:
+    """
+    Test that for assets in the same collection only one file is saved and
+    is later used by all the assets in the same collection
+    """
+    dai_op = Asset('eip155:10/erc20:0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1')
+    times_api_was_queried = 0
+
+    # remove icon file in case it was added by another test
+    icon_path = icon_manager.iconfile_path(A_DAI)
```

Comment:
will this affect production data? No right? This should go under "test_data" directory? Perhaps make sure by having the data fixture directory point to a random directory?

Reply:
used a clean cache directory